db/migrate/20181010151538_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,56 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    
+    create_table(:users) do |t|
+      ## Required
+      t.string :provider, :null => false, :default => "email"
```

Comment:
remove provider and uid

Reply:
toca tirar rollback? tengo la base de datos con lo ultimo que he estado haciendo

Reply:
<img width="815" alt="screen shot 2018-10-12 at 11 00 35 am" src="https://user-images.githubusercontent.com/43754771/46880466-55413180-ce0e-11e8-8f0c-476f86daae7b.png">

Al quitar esas dos columnas, no puedo crear un User ya que me pide, `provider`